### ISSUE TITLE
Instance Block Severity between Silence and Suspend "Restrictive" 

### ISSUE THREAD

**Issue by shleeable** [mastodon/mastodon#20549]
### Pitch

We've reached a point of growth where users are asking us "why can't I contact X on suspended instance Y".... but the moderation team has voted to keep instance Y in the suspend due to not being moderated or whatever safety reason.

Silence is too open, and suspend is too harsh. I'd like a "Limited" or "Restrictive" mode to place risky instances into.

Something like... `Suspend, but users can poke holes in the suspend on a user by user basis only for themselves.`.

### Motivation

People really like talking to their friends, but risky instances behavior is being proactively suspended.

**Comment by shleeable** [comment]
So. I've spoken to a few instance admins about this.. We're looking for something stronger than limit, but not as absolute as suspend.

"this users instance has been marked as restricted by aus.social, if you wish to communicate with this user, you can opt-in to communication with this user only on this restricted instance" Press button to allow user X to communicate with yourself.

**Comment by ekovac** [comment]
Agreed, it also seems like the docs https://docs.joinmastodon.org/admin/moderation/ don't really explain with full clarity what "silence" actually means. If it does I can't immediately find it.

**Comment by trwnh** [comment]
This sounds like the existing limit/silence option. Which specifically missing characteristics are you suggesting?
